### PR TITLE
Add some more examples to function documentation

### PR DESCRIPTION
**Pull request by rusty-snake**

### CODE REVIEW

**Comment by ManaSugi** [comment]
Thank you for the PR. 
Could you update `CHANGELOG` too? I think it is better to add this change to `CHANGELOG`  in order to inform users of this examples.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 ### Added
+- some more examples to function documentation.
```
Comment (ManaSugi) [inline CHANGELOG.md:10]: Nit: `some` -> `Some`

Comment (rusty-snake) [reply CHANGELOG.md:10]: fixed

Comment (ManaSugi) [reply CHANGELOG.md:10]: Oh, I cannot see the change.

Comment (rusty-snake) [reply CHANGELOG.md]: Now it should be better.

**Comment by ManaSugi** [comment]
LGTM, Thank you! I'll release v0.2.2 (I hope the documentations will go well).

**Review by ManaSugi — APPROVED** [review]

**Pull request merged**